pcli/src/opt.rs

```diff
@@ -61,6 +74,19 @@ pub enum WalletCmd {
     Delete,
 }
 
+impl WalletCmd {
+    /// Determine if this command requires a network sync before it executes.
+    pub fn needs_sync(&self) -> bool {
+        match self {
```

Comment:
This match statement is deliberately verbose because it means that the addition of another subcommand forces future programmers to consider whether or not it should require synchronization beforehand, rather than automatically defaulting it to `false`.